.github/workflows/presubmit.yml

```diff
@@ -10,6 +10,9 @@ jobs:
         os: [ubuntu-latest, macos-latest, windows-latest]
     steps:
       - uses: actions/checkout@v2
+      - uses: lukka/get-cmake@latest
+        with:
+          cmakeVersion: 3.14.0
```

Comment:
Explicitly test the advertised minimum:
https://discourse.cmake.org/t/check-your-cmake-minimum-required-with-get-cmake/7257